source/atlas-cli-deploy-docker.txt

```diff
@@ -309,7 +273,89 @@ of Docker Compose.
 
       .. code-block:: sh
 
-         mongosh mongodb://root:{your-password}@localhost:27017/?directConnection=true
+         mongosh "mongodb://user:pass@localhost:27019/?directConnection=true"
+
+Generate a List of Dependencies
+-------------------------------
+
+You can generate a list of the dependances for the 
+``mongodb/mongodb-atlas-local`` Docker image.
+
+.. procedure::
+   :style: normal
+
+   .. step:: Install syft.
+
+      **Example:**
+
+      .. code-block:: sh
+
+         brew install syft
+
+      To learn more, see the :github:`syft README 
+      </anchore/syft/blob/main/README.md>`.
```

Comment:
I would move these two lines to be directly after line 281 about the intro to the procedure.

Reply:
@JuliaMongo, I prefer it here. If they can use, brew, they can ignore.